src/cljs/rems/navbar.cljs

```diff
@@ -80,13 +84,17 @@
      [language-switcher]]))
 
 (defn navbar-normal [identity]
-  [:nav.navbar-flex
-   [:div.navbar.navbar-expand-sm.flex-fill
-    [:button.navbar-toggler
-     {:type :button :data-toggle "collapse" :data-target "#small-navbar"}
-     "\u2630"]
-    [navbar-items :div#big-navbar.collapse.navbar-collapse.mr-3 identity]]
-   [:div.navbar [user-widget (:user identity)]]])
+  (let [theme @(rf/subscribe [:theme])]
+    [:nav.navbar-flex
+     [:div.navbar.navbar-expand-sm.flex-fill
+      (if (:upper-left-logo theme)
+        [logo (:upper-left-logo theme)]
+        nil)
```

Comment:
This can be written with `when` as you don't have the else clause.

Reply:
true. Good point, will fix